plugin/EstateList.php

```diff
@@ -723,4 +727,12 @@ public function setFormatOutput(bool $formatOutput)
 	/** @return EstateListEnvironment */
 	public function getEnvironment(): EstateListEnvironment
 		{ return $this->_pEnvironment; }
+
+	/** @return string */
+	public function getReferenz(): string
+	{ return $this->_referenz; }
+
+	/** @param string $referenz */
+	public function setReferenz(string $referenz)
```

Comment:
Is there a better solution than adding this new property?

Reply:
@jayay What do you think about adding a new element to the array in the estateIterator function like this?
![image](https://user-images.githubusercontent.com/72601287/146480311-77c5386e-1ae2-450d-a60e-447edae5e071.png)

Reply:
That looks cleaner, writing `$pArrayContainer['referenz'] = $recordRaw['referenz'] ?? null` would make it a little bit safer, but your proposal could break sites. How about a public interface in `EstateList` to retrieve the `$_recordsRaw`? It should return the raw values inside an `ArrayContainerEscape` as well. If the original value (without escaping, which in turn makes it a string) is needed, it has to be fetched using `$pArrayContainer->getValueRaw()`.